sklearn/cluster/_agglomerative.py

```diff
@@ -378,35 +379,35 @@ def linkage_tree(
     Parameters
     ----------
     X : array-like of shape (n_samples, n_features)
-        feature matrix representing n_samples samples to be clustered
+        feature matrix representing `n_samples` samples to be clustered.
 
     connectivity : sparse matrix, default=None
         connectivity matrix. Defines for each sample the neighboring samples
         following a given structure of the data. The matrix is assumed to
         be symmetric and only the upper triangular half is used.
-        Default is None, i.e, the Ward algorithm is unstructured.
+        Default is `None`, i.e, the Ward algorithm is unstructured.
 
     n_clusters : int, default=None
-        Stop early the construction of the tree at n_clusters. This is
+        Stop early the construction of the tree at `n_clusters`. This is
         useful to decrease computation time if the number of clusters is
         not small compared to the number of samples. In this case, the
         complete tree is not computed, thus the 'children' output is of
         limited use, and the 'parents' output should rather be used.
         This option is valid only when specifying a connectivity matrix.
 
-    linkage : {"average", "complete", "single"}, default="complete"
+    linkage : {'average', 'complete', 'single'}, default='complete'
```

Comment:
I would keep the double quotes since this is also the style used by black in the code.